### PR TITLE
Setting CRTSimT0Offset to 1.6ms in MC CAFMaker

### PR DESCRIPTION
**Pull request by jedori0228**
This feature was introduced in https://github.com/SBNSoftware/icaruscode/pull/384 to [release/SBN2022A](https://github.com/SBNSoftware/icaruscode/tree/release/SBN2022A), but never been propagated to develop branch. This is to incorporate with the intrinsic "shift" in CRT simhit timimgs; They all starts from "0", but those zeros represents -1.6ms before the beam gate. I put two references where this value is defined.

### CODE REVIEW

**Comment by mmrosenberg** [comment]
trigger build

File: fcl/caf/cafmakerjob_icarus.fcl [hunk 1]
```diff
@@ -47,5 +47,9 @@ physics:
   end_paths:     [ stream1 ]
 }
 
+# MCT0Offset for MC ()
+# ref1 : https://github.com/SBNSoftware/icaruscode/blob/v09_37_02_01/icaruscode/CRT/crtsimmodules_icarus.fcl#L11
+# ref2 : https://github.com/SBNSoftware/icaruscode/blob/v09_64_01/icaruscode/CRT/crttruehitproducer.fcl#L8
+physics.producers.cafmaker.CRTSimT0Offset: 1600000
```
Comment (PetrilloAtWork) [inline fcl/caf/cafmakerjob_icarus.fcl:53]: Proposed alternative:
* `#include "crtsimmodules_icarus.fcl"`
* ```suggestion
  physics.producers.cafmaker.CRTSimT0Offset: @local::standard_icarus_crtsimalg.GlobalT0Offset
   ```

**Review by PetrilloAtWork — COMMENTED** [review]
I am proposing an alternative for your consideration, which links directly the parameter to the value in its standard configuration.
The request is good as is, if you choose to pass the suggesion.

File: fcl/caf/cafmakerjob_icarus.fcl (hunk 1 above)
Comment (jedori0228) [reply fcl/caf/cafmakerjob_icarus.fcl]: Thanks Gianluca, your suggestion is way better! I updated it as you recommended, and checked correct number is put.

**Review by francescopoppi — APPROVED** [review]

**Comment by mmrosenberg** [comment]
trigger build

**Pull request merged**